tests/framework_cherrypy/tox.ini

```
@@ -0,0 +1,29 @@
+[tox]
+setupdir = {toxinidir}/../..
+envlist =
+    {py35,py36,py37,py38,py39,pypy3}-CherryPy14,
+    {py35,py36,py37}-CherryPy0302,
+    pypy3-CherryPy0303
+
+[pytest]
+usefixtures = session_initialization requires_data_collector
+
+[testenv]
+deps =
+    routes
+    CherryPy0302: CherryPy<3.3.0
+    CherryPy0303: CherryPy<3.4.0
+    CherryPy14: CherryPy<15.0
```

```suggestion
    {py35,py36,py37,py38,py39,pypy3}-CherryPy18,
    {py35,py36,py37}-CherryPy0302,
    pypy3-CherryPy0303

[pytest]
usefixtures = session_initialization requires_data_collector

[testenv]
deps =
    routes
    CherryPy0302: CherryPy<3.3.0
    CherryPy0303: CherryPy<3.4.0
    CherryPy18: CherryPy<19.0
```

After testing coverage, there is no functional difference between our tests on v18 and v14